taker-electron/src/main/menu.ts

```diff
@@ -0,0 +1,257 @@
+            ],
+        };
+
+        const subMenuView = process.env.NODE_ENV === "development"
+                || process.env.DEBUG_PROD === "true"
+            ? subMenuViewDev
+            : subMenuViewProd;
+
+        return [subMenuAbout, subMenuEdit, subMenuView, subMenuWindow, subMenuHelp];
+    }
+
+    buildDefaultTemplate() {
+        const templateDefault = [
+            {
+                label: "&File",
+                submenu: [
+                    {
+                        label: "&Open",
+                        accelerator: "Ctrl+O",
+                    },
+                    {
+                        label: "&Close",
+                        accelerator: "Ctrl+W",
+                        click: () => {
+                            this.mainWindow.close();
+                        },
+                    },
+                ],
+            },
+            {
+                label: "&View",
+                submenu: process.env.NODE_ENV === "development"
+                        || process.env.DEBUG_PROD === "true"
+                    ? [
+                        {
+                            label: "&Reload",
+                            accelerator: "Ctrl+R",
+                            click: () => {
+                                this.mainWindow.webContents.reload();
+                            },
+                        },
+                        {
+                            label: "Toggle &Full Screen",
+                            accelerator: "F11",
+                            click: () => {
+                                this.mainWindow.setFullScreen(
+                                    !this.mainWindow.isFullScreen(),
+                                );
+                            },
+                        },
+                        {
+                            label: "Toggle &Developer Tools",
+                            accelerator: "Alt+Ctrl+I",
+                            click: () => {
+                                this.mainWindow.webContents.toggleDevTools();
+                            },
+                        },
+                    ]
+                    : [
+                        {
+                            label: "Toggle &Full Screen",
+                            accelerator: "F11",
+                            click: () => {
+                                this.mainWindow.setFullScreen(
+                                    !this.mainWindow.isFullScreen(),
+                                );
+                            },
+                        },
+                    ],
+            },
+            {
+                label: "Help",
+                submenu: [
+                    {
+                        label: "Learn More",
+                        click() {
+                            shell.openExternal("https://itchysats.network");
+                        },
+                    },
+                    {
+                        label: "FAQ",
```

Comment:
super cool to have the FAQ link there!!! 💯

other ideas for additional items:
-  `Join  Community` with the telegram group link
- `Ask for Help` with the link to github discussions